### DISCUSSION TITLE
Quickdraw achievement earned after closing my own topic

### DISCUSSION THREAD

**Discussion by emilrueh** [community/community#40145] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I earned the achievement 'Quickdraw: Topic closed within 5 minutes of opening' by closing my own topic.
Perhaps as it says Beta that would be a feature to have available for closing threads only after answered by someone else to avoid exploiting it.

**Comment by ossanna16** [comment]
Hi there @emilrueh and welcome to our community! Thank you for the feedback!

**Comment by StoiaCode** [comment]
I second this, closing my own PR a second after opening because I noticed a mistake shouldnt give me a reward 😆

**Comment by geekceo** [comment]
Nice achievement - less work, more beautiful trinkets  😆😆
